tasks/68_drop-elements/drop-elements.js

```diff
@@ -0,0 +1,8 @@
+export function dropElements(arr, func) {
+  const cloneArr = [...arr];
+  for (let i = 0; i < arr.length; i++) {
+    if (func(arr[i])) return cloneArr;
```

Comment:
Please try to remain consistency and use { } for if/else statement.

```suggestion
    if (func(arr[i])) {
      return cloneArr;
    }
```